src/wallet/rpcwallet.cpp

```diff
@@ -4185,7 +4188,7 @@ UniValue z_mergetoaddress(const UniValue& params, bool fHelp)
             "  \"mergingTransparentValue\": xxx      (numeric) Value of UTXOs being merged.\n"
             "  \"mergingNotes\": xxx                 (numeric) Number of notes being merged.\n"
```

Comment:
This is really nitpicking but there are so many differences in how a numeric output is displayed in this file it's hard to know if there is intent in how the xxx is presented. Other numeric output in this file (say `z_listunspent`) is also shown as n (which presumably is an integer?) but also xxxxx.

Reply:
"numeric" seems to sometimes mean integer and sometimes decimal. In some of the original bitcoin rpcs it looks like they are using *x.xxx* for decimal and *n* for integer. I think I like that convention, but I am just going to leave this for the time being.

Reply:
Agreed. `getwalletinfo` for example has about every variation imaginable 😄